### ISSUE TITLE
Log moment that user joined the channel

### ISSUE THREAD

**Issue by martinbeentjes** [termoose/irccloud#1]
Hi, thanks for this wonderful piece of software! 

It would be a nice feature if the join messages of the user using the tool is also shown in the buffer. Now I have to dig into the timestamps to determine when I joined again.

Is this possible using the IRCCloud API? If so, how can I/we contribute?

**Comment by termoose** [comment]
Hi not sure what you mean exactly, but sounds like you'd like something like this
```
[19:57]  -> martinbeen joined (some@hostname)
```
at the start of the buffer for when you joined? In that case it should be easy to add. Thanks for opening the first feature request!

**Comment by martinbeentjes** [comment]
Yes, that is exactly what I meant. Sorry for making it hard to understand.

Great work on this little tool!

**Comment by c99koder** [comment]
`you_joined_channel` is the event in our API for when the user joins a channel
